### ISSUE TITLE
Inventory screen expansion?

### ISSUE THREAD

**Issue by mrtree3** [nipkownix/re4_tweaks#232]
### Describe the feature you'd like to see added

Hello, would it be possible to shrink inventory items so we can have more room for item carrying?
Or perhaps an extra second layer of inventory space? Thanks for all that's being done here!

**Comment by Mister-Curious** [comment]
In theory it is possible to edit all case items so that they would be 1x1 slots, but this would take some tedious editing of the exe and all you could be left with are shapes the size of egg items. Do you really want all your inventory items to be 1x1?

While I think adding a secondary layer of inventory items is a novel idea, I do believe that making this happen is well beyond the skills and knowledge of anyone I know that mods this game. 

You would be better off using razor's trainer and simply saving items to file and loading them in that way.

**Comment by mrtree3** [comment]
Fair enough, I had no idea if it was even possible lol the extra second layer would've been nice in the largest briefcase though, ah well.
